### ISSUE TITLE
Add support for new 'Access Worldpay' API

### ISSUE THREAD

**Issue by lukeholder** [thephpleague/omnipay-worldpay#38]
There is a new REST API the 'next generation API':

https://developer.worldpay.com/docs/access-worldpay/get-started

Guessing it should be added as a new gateway.

**Comment by jamieburchell** [comment]
It seems that WorldPay have buried the documentation for what must now be the "legacy" WorldPay Online Payments API - as it's not linked to from the API page.

If this legacy integration doesn't support SCA, I think this library will stop working?

**Comment by timothymarois** [comment]
Any word on this? Sounds like this isn't updated to use the new WorldPay API

**Comment by bencroker** [comment]
There's a WIP at https://github.com/putyourlightson/omnipay-worldpay-access

**Comment by timothymarois** [comment]
@bencroker Thanks! Very little documentation on these SDKs from what I can see. Do they all implement the same omnipay authorization headers or something because they don't really describe how to use them aside from linking to omnipay.

**Comment by bencroker** [comment]
You're right @timothymarois, I developed this for a client and haven't taken the time to document it, but the process of usage is generally the same for all Omnipay drivers.

**Comment by leeovery** [comment]
Hey @bencroker 

Apologies for the non-issue related comment but hopefully this could help others too...

The new world pay api plug-in you linked to above... is it production ready and fully featured?

I have a project to migrate an existing site from the old world pay to this new one. The old one is being shut down in October so time is off the essence! 

Thanks either way!!
Lee

**Comment by bencroker** [comment]
@leeovery it is being used in production, but as you can see it's been left alone for 17 months so very likely could use some testing and compatiblity for Craft 4 too. If you have an urgent need then get in touch and we can discuss what it might cost to get it production-ready for you.

**Comment by leeovery** [comment]
@bencroker Thanks. The client isn't actually using Craft at all. It's a custom built crm with worldpay used for payments etc. 

Thanks for your reply. I'll take a look and see what's what.
